<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>start_form</class>
 <widget class="QWidget" name="start_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Expression Recognizer</string>
  </property>
  <widget class="QFrame" name="horizontalFrame">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>500</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>60</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="image_BTN">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>IMAGE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="video_BTN">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>VIDEO</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="horizontalFrame">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>400</y>
     <width>500</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>60</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="settings_BTN">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="quit_BTN">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
